docs/reference/ilm/actions/ilm-rollover.asciidoc

```diff
@@ -4,7 +4,7 @@
 
 Phases allowed: hot.
 
-Rolls over a target to a new index when the existing index meets one of the rollover conditions.
+Rolls over a target to a new index when the existing index meets one or more of the rollover conditions.
```

Comment:
Maybe I'm misunderstanding the question but if any conditions (one or more) are met the rollover is performed (a new index is created). The outcome is that a new index is created.

Reply:
@andreidan Yes, but we have quite often feedback around exactly this topic that people want to understand which one will win, so it would be cool if we had a small sentence that describes how either to find that out or to describe the principle of this is a question of order of what wins first max_age, max_docs or max_size or if it is just a simple logical or.

Reply:
@111andre111 thanks for the clarification. I'm a bit out of ideas on how to state that there is no priority of any sort for the conditions (beyond the already suggested `one or more of the rollover conditions` in this PR). 
Please feel free to suggest something concrete that'll improve the docs in this PR or a different one.

> what happens if multiple conditions theoretically might be applied at the point in time which one wins?

Furthermore, I think we shouldn't use terms like "applied" and "winning" when it comes to conditions, as that implies side effects and possibly a ranking - I did a quick grep in our docs and we don't use the apply/applied terminology in our condition description. Conditions are met (or not), and based on at least one condition being met the rollover is executed (the indices stats API can be used to track the size and number of docs in an index, while the age of the index is available in the `_ilm/explain` API)